src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@1,400;1,700&family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+
+  font-family: inherit;
+  font-size: 12px;
+  text-transform: uppercase;
+  padding: 0 50px;
+  margin: 0;
+
+  background: white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.link_box {
+  display: flex;
+  align-items: center;
+}
+
+.link_list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.list_item {
+  margin-left: 20px;
+}
```

Comment:
And what about the first element? This thing can break tests

Reply:
Done, and yes, fixing that mistake helped me to pass a few more test without difficulties